mrtt-ui/src/views/SiteQuestionsOverview.js

```diff
@@ -0,0 +1,145 @@
+import { Link, useParams } from 'react-router-dom'
+import { Settings } from '@mui/icons-material'
+import { Stack } from '@mui/material'
+import { styled } from '@mui/system'
+import { toast } from 'react-toastify'
+import axios from 'axios'
+import React, { useEffect, useState } from 'react'
+
+import { H4, H5Uppercase, SmallUpperCase, TinyUpperCase } from '../styles/typography'
+import { PaddedPageSection, PaddedPageTopSection, RowSpaceBetween } from '../styles/containers'
+import { TableAlertnatingRows } from '../styles/table'
+import ItemDoesntExist from '../components/ItemDoesntExist'
+import language from '../language'
+import LoadingIndicator from '../components/LoadingIndicator'
+import theme from '../styles/theme'
+
+const pageLanguage = language.pages.siteQuestionsOverview
+
+const SettingsLinkWrapper = styled(Link)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  text-decoration: none;
+  color: ${theme.color.text};
+`
+const TinyTd = styled('td')`
+  text-align: center;
+`
+const GreedyTh = styled('th')`
+  width: 100%;
+`
+const SettingsLink = (props) => (
+  <SettingsLinkWrapper {...props}>
+    <Settings />
+    <TinyUpperCase>{pageLanguage.settings}</TinyUpperCase>
+  </SettingsLinkWrapper>
+)
+
+const SiteOverview = () => {
+  const [doesSiteExist, setDoesSiteExist] = useState(true)
+  const [isLoading, setIsLoading] = useState(true)
+  const [site, setSite] = useState()
+  const { siteId } = useParams()
+
+  useEffect(
+    function loadDataFromServer() {
+      if (siteId) {
+        const sitesDataUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}`
+
+        Promise.all([axios.get(sitesDataUrl)])
```

Comment:
made this promise.all because in the next tickets I will be loading data from all the forms to display form progress